gin/pom.xml

```diff
@@ -41,6 +41,7 @@
         <dependency>
             <groupId>org.jboss.gwt.elemento</groupId>
             <artifactId>elemento-core</artifactId>
+            <type>gwt-lib</type>
```

Comment:
this is not required, but as you are defining the version in the parent pom, this makes this version used by this dependency. You can instead remove the type and use <version>${project.version}</version>.